src/main/java/com/google/sps/model/ActionableMessage.java

```diff
@@ -44,9 +47,13 @@ public String getSubject() {
     return subject;
   }
 
+  public String getSender() {
+    return sender;
+  }
+
   @Override
   public int hashCode() {
-    return new HashCodeBuilder(7, 17).append(id).append(subject).toHashCode();
+    return new HashCodeBuilder(7, 17).append(id).append(subject).append(sender).toHashCode();
```

Comment:
nit: why not the default `HashCodeBuilder` constructor?

Reply:
+1 to using the default constructor

Reply:
Agreed - thanks!